pkgs/development/libraries/gstreamer/gstreamermm/default.nix

```diff
@@ -20,9 +16,16 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  passthru = {
+    updateScript = gnome3.updateScript {
+      packageName = pname;
+      versionPolicy = "none";
```

Comment:
This most definitely uses GNOME version policy, as both GStreamer and GLibmm do.

Reply:
I saw that they marked a lot of even versions as unstable.
https://gitlab.gnome.org/GNOME/gstreamermm/blob/master/NEWS

Reply:
Wow. Could you add a comment like “unpredictable version stability” then?